fiftyone/server/routes/embeddings.py

```diff
@@ -252,12 +258,6 @@ async def post(self, request: Request, data: dict) -> dict:
 ]
 
 
-def _load_dataset(dataset_name):
```

Comment:
Note how the current implementation is stored a *permanent* reference to datasets that are used in the embeddings routes. Some version of this PR needs to be checked in to resolve that!